contracts/seed/Seed.sol

```diff
@@ -217,9 +217,12 @@ contract Seed {
     /**
       * @dev         Returns funding tokens to user.
     */
-    function retrieveFundingTokens() public protected checked beforeMinimumReached {
+    function retrieveFundingTokens() public protected /* checked */ beforeMinimumReached {
```

Comment:
pls remove the comment

Reply:
Yes, waiting for tests to run successfully.